packages/json_test.go

```diff
@@ -70,7 +70,7 @@ func TestWatch(t *testing.T) {
 	if err := ioutil.WriteFile(testFile, data, 0644); err != nil {
 		t.Fatalf("Error writing to file %s: %s", testFile, err)
 	}
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(1000 * time.Millisecond)
```

Comment:
Is this necessary? I really would like the tests to run as fast as possible, within reason.